### ISSUE TITLE
Fetch data for most watched page & render most watched component

### ISSUE THREAD

**Issue by jroebu14** [bbc/simorgh#7641]
Example map: [Miro](https://miro.com/app/board/o9J_kvZ2wMk=/?moveToWidget=3074457349544789098&cot=12)
Some analysis of existing routes: [Dropbox Paper](https://paper.dropbox.com/doc/Analysis-Most-Watched--A6ccw62pyGe4I0pgfBjDw8iDAg-TK3nz2VKkDvUNC8P1rnh7) 
Jira ticket: [Jira](https://jira.dev.bbc.co.uk/browse/WSMEDIA-234)
Design: [Zeplin](https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/5f22de67d3d1fd1f74e4cda1)

Blocked until the basic page routing is done in https://github.com/bbc/simorgh/issues/7640

**Is your feature request related to a problem? Please describe.**
The user should be able to view the most watched component when visiting e.g. https://www.bbc.com/afaanoromoo/media/video

**Describe the solution you'd like**

- add `getInitialData` function in route module to fetch most watched data and pass to page component (route must have been set up in https://github.com/bbc/simorgh/issues/7640)

Add Most watched component to page
 - https://github.com/bbc/simorgh/blob/latest/src/app/containers/MostWatched/index.jsx
- The Most watched component requires some changes
  - should not display the h2 most watched title
  - add logic so the promo headline is an h2 on most watched pages (not an h3 as it currently is when not on most watched page)

Question: How many items do we show? Currently in PAL there are 20

**A11y Acceptance Criteria**
Most Watched should be within the main container and the h1 should be visually displayed as ‘Most Watched'

The skip to content link should anchor here (note there should not be an h2 for most watched or any other regions for most watched within the main region)

**Skip to content link**
Given I use a keyboard only
When I navigate to the banner landmark by pressing the TAB key
Then the ‘skip to content’ link must be the first piece of content to receive focus after the brand logo
And the ‘skip to content’ link must be visually displayed on focus
And this must include the browser default focus outline styling

Given I use a screen reader
When I activate the ‘skip to content’ link
Then the h1 in the main region should be announced
And after this announcement, I must be able to navigate to the next piece of content below the h1

Given there is a ‘skip to content’ link
When I activate the ‘skip to content’ link
Then the url should be user friendly and descriptive of it’s destination e.g. #content

**List of promos (within section)**
Given I use a screen reader
When I view a section with more than one promo 
And I navigate to the first promo
Then it should be announced as a list item within a list

**Promo image**
StoryPromo - Psammead unit test
https://github.com/bbc/psammead/issues/541

Given I use a screen reader
When I view a promo with an image 
And I navigate to the promo image
Then the image alt text must be announced
Promo h2

**StoryPromo - Psammead unit test**
https://github.com/bbc/psammead/issues/541

Given I use a screen reader
When I navigate to a promo
Then this must have one heading level 2
And must be announced as a link

NB: See also full promo a11y a/c: https://docs.google.com/document/d/1wC9vTLCt5RwfKIHn7spWsmwvt7INl4w1Pfrl7jD8iZA/edit#heading=h.myflp6au8u7p

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

Blocked by https://github.com/bbc/simorgh/issues/7640

**Comment by greenc05** [comment]
Should we also be making the change of no section/region for the Most watched component in this Issue @jroebu14?

"(note there should not be an h2 for most watched **or any other regions for most watched within the main region**)"

**Comment by greenc05** [comment]
Think we are also missing changing the ul to an ol? (This has been added to another Issue now https://github.com/bbc/simorgh/issues/7707)
